packages/backend/Dockerfile

```diff
@@ -1,16 +1,25 @@
-FROM node:14-buster
+# This dockerfile builds an image for the backend package.
+# It should be executed with the root of the repo as docker context.
+#
+# Before building this image, be sure to have run the following commands in the repo root:
+#
+# yarn install
+# yarn tsc
+# yarn build
```

Comment:
Add a line high up on how to actually invoke it too (the yarn command), that's the most interesting rather than how to issue it "by hand"

Reply:
Invoke what? O_o

Reply:
Essentially if I browse the source code and I stumble on this file, my most pressing ask isn't "how do I form a large bash command that successfully uses this file", it's "oh so this one is used by yarn blah:bleh in the root (which forms that bash command)" :)

Reply:
There's no command that runs those 3 for you, don't think we should add it either tbh

Reply:
I don't mean those three. I'm not being clear and I should have placed this comment higher up. What I'm referring to is the text "It should be executed with the root of the repo as docker context." mostly. It would just be nice to have a line saying "This Dockerfile is used by the yarn blah command to build a backend image. It is run with the root of the repo as the docker context." which explains how it is invoked in reality, rather than as if the user is expected to invoke it in a specific way

Reply:
Ah alright, mentioned the `yarn build-image` command